app/src/App/routes/SurgeRoutes.tsx

```diff
@@ -839,6 +839,160 @@ const surgeCatalogueWash = customWrapRoute({
     },
 });
 
+const surgeCatalogueCMR = customWrapRoute({
+    parent: surgeCatalogueLayout,
+    path: 'other/civil-military-relations',
+    component: {
+        render: () => import('#views/SurgeCatalogueOtherCivilMilitaryRelations'),
+        props: {},
+    },
+    wrapperComponent: Auth,
+    context: {
+        title: 'Civil Military Relations (CMR)',
+        visibility: 'anything',
+    },
+});
+
+const surgeCatalogueDDR = customWrapRoute({
```

Comment:
Lets make the variable name consistent with the view name.
```suggestion
const surgeCatalogueOtherDisasterRiskReduction = customWrapRoute({
```